dashboard/src/actions/authActions.js

```diff
@@ -2,114 +2,58 @@ import * as APP_ROUTES from "utils/routeConstants";
 import * as CONSTANTS from "../assets/constants/authConstants";
 import * as TYPES from "./types";
 
-import API from "../utils/axiosInstance";
 import Cookies from "js-cookie";
 import { SUCCESS } from "assets/constants/overviewConstants";
 import { showToast } from "actions/toastActions";
-import { uid } from "../utils/helper";
 
 
-// Create an Authentication Request
-export const authenticationRequest = () => async (dispatch, getState) => {
-    try {
-      const endpoints = getState().apiEndpoint.endpoints;
-      const oidcServer = endpoints.openid.server;
-      const oidcRealm = endpoints.openid.realm;
-      const oidcClient = endpoints.openid.client;
-      // URI parameters ref: https://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint
-      // Refer Step 3 of pbench/docs/user_authentication/third_party_token_management.md
-      const uri = `${oidcServer}/realms/${oidcRealm}/protocol/openid-connect/auth`;
-      const queryParams = [
-        'client_id=' + oidcClient,
-        'response_type=code',
-        'redirect_uri=' + window.location.href.split('?')[0],
-        'scope=profile',
-        'prompt=login',
-        'max_age=120'
-      ];
-      window.location.href = uri + '?' + queryParams.join('&');
-    } catch (error) {
-      const alerts = getState().userAuth.alerts;
-      dispatch(error?.response
-        ? toggleLoginBtn(true)
-        : { type: TYPES.OPENID_ERROR }
-      );
-      const alert = {
-          title: error?.response ? error.response.data?.message : error?.message,
-          key: uid(),
-      };
-      dispatch({
-        type: TYPES.USER_NOTION_ALERTS,
-        payload: [...alerts, alert],
-      });
-      dispatch({ type: TYPES.COMPLETED });
+/**
+ * Loop to check if the endpoints are loaded.
+ * @param {getState} getState object.
+ * @return {promise} promise object
+ */
+export function waitForKeycloak (getState) {
+  const waitStart = Date.now();
+  const maxWait = 10000;  // Milliseconds
```

Comment:
Why? It isn't used by anyone else.

Reply:
It's not about where the symbol is referenced, it's about how we (as humans) access the definition.

When we want to configure the behaviors of the Dashboard, it would be nice if we could go to a "definitions file" or a "configuration file" or some similarly centralized source, rather than having to chase down all the spots in the code where these settings are made.

Reply:
Fixed.

Reply:
I [commented](https://github.com/distributed-system-analysis/pbench/pull/3250#discussion_r1147963741) on the new location; we can follow up there.